src/components/Sidebar/ActionNode.tsx

```diff
@@ -1,49 +1,212 @@
 import React, { FC, memo, useCallback, ChangeEvent } from 'react';
 import { SidebarButton } from '.';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faPlus } from '@fortawesome/free-solid-svg-icons';
-
+import { faPlus, faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import { setStateAction } from 'utils/builder';
 import DropDown from 'components/elements/DropDown';
-import { Pathway, State } from 'pathways-model';
+import Input from 'components/elements/Input';
+import { Pathway, GuidanceState, Action } from 'pathways-model';
 import useStyles from './styles';
+import shortid from 'shortid';
 
 const nodeTypeOptions = [
   { label: 'Action', value: 'action' },
   { label: 'Branch', value: 'branch' }
 ];
 
+const actionTypeOptions = [
+  { label: 'Medication Request', value: 'Medication Request' },
+  { label: 'Service Request', value: 'Service Request' },
+  { label: 'Careplan', value: 'Careplan' }
```

Comment:
The values here are being used in the `resource` but are not the correct fhirnames
```suggestion
  { label: 'Medication Request', value: 'MedicationRequest' },
  { label: 'Service Request', value: 'ServiceRequest' },
  { label: 'Careplan', value: 'CarePlan' }
```